station-renderer-arsenic/src/main/java/net/modificationstation/stationapi/impl/client/arsenic/renderer/render/BakedModelRendererImpl.java

```diff
@@ -421,17 +426,12 @@ protected void renderGuiItemModel(ItemInstance stack, int x, int y, BakedModel m
         matrices.translate(8.0, 8.0, 0.0);
         matrices.scale(1.0f, -1.0f, 1.0f);
         matrices.scale(16.0f, 16.0f, 16.0f);
-        boolean bl = !model.isSideLit();
-        if (bl) {
-            RenderHelper.disableLighting();
-        }
+
         Tessellator.INSTANCE.start();
         this.renderItem(stack, ModelTransformation.Mode.GUI, false, matrices, StationTessellator.get(Tessellator.INSTANCE), -1/*LightmapTextureManager.MAX_LIGHT_COORDINATE*/, -1/*OverlayTexture.DEFAULT_UV*/, model);
         Tessellator.INSTANCE.draw();
         GL11.glEnable(GL11.GL_DEPTH_TEST);
-        if (bl) {
-            RenderHelper.enableLighting();
```

Comment:
That seems dangerous. Are you sure there are no side effects of not reenabling lighting on exit?

Reply:
Lighting needs to be set based on item type, true if block-item, false if normal item. If lighting needs to be disabled post-gui rendering, then it should be done there, but I haven't seen any side-affects yet